modules/kallistobustools/count/main.nf

```diff
@@ -36,12 +38,11 @@ process KALLISTOBUSTOOLS_COUNT {
         -g $t2g \\
         $cdna \\
         $introns \\
-        --workflow $workflow_mode \\
+        --workflow $sc_workflow \\
```

Comment:
Should we add a `-m <memory>` flag. 

according to the docs it supports the giga notation: `4G`